src/OrleansRuntime/LogViews/GrainStateWithMetaData.cs

```diff
@@ -30,15 +36,27 @@ object IGrainState.State
             }
         }
 
+        /// <summary>
+        /// Constructor with TView
+        /// </summary>
+        /// <param name="initialview"></param>
```

Comment:
please either add a comment for the `param` and `returns` elements, or remove them entirely (which gets rid of the warning too). Having empty comments is just noise